docs/extra/uninstall.md

```diff
@@ -1,16 +1,28 @@
 # Uninstall
 
-To ensure a clean removal you should uninstall Codacy before destroying the cluster. To do so run:
+To ensure a clean removal you should uninstall Codacy before destroying the cluster.
+To do so run:
 
 ```bash
 helm del --purge codacy
-kubectl delete pvc -n codacy $(kubectl get pvc -n codacy -o jsonpath='{.items[*].metadata.name}') &
-kubectl delete pods -n codacy $(kubectl get pods -n codacy -o jsonpath='{.items[*].metadata.name}') --force --grace-period=0
+kubectl -n codacy delete --all pod &
+kubectl -n codacy delete --all pvc &
+kubectl -n codacy delete --all pv  &
+kubectl -n codacy delete --all job &
+sleep 5
+kubectl -n codacy patch pvc -p '{"metadata":{"finalizers":null}}' $(kubectl -n codacy get pvc -o jsonpath='{.items[*].metadata.name}')
+kubectl -n codacy patch pv -p '{"metadata":{"finalizers":null}}' $(kubectl -n codacy get pv -o jsonpath='{.items[*].metadata.name}')
+sleep 5
+kubectl -n codacy delete pod $(kubectl -n codacy get pod -o jsonpath='{.items[*].metadata.name}') --force --grace-period=0
+kubectl -n codacy get pod &
+kubectl -n codacy get pvc &
+kubectl -n codacy get pv  &
+kubectl -n codacy get job &
 ```
 
 Note that the deletion of `pvc`s in the above command has to run in the background
 due to a cyclic dependency in one of the components. If you are unsure of these
-commands' effect please run the each of the `bash` subcommands and validate their output, viz.
+commands' effect please run the each of the `bash` subcommands and validate their output.
```

Comment:
Suggestion and fix typo:

"If you are unsure of the effects of these commands please run each of the `bash` subcommands and validate their output."